plugins/microsoft_atp/plugin.spec.yaml

```diff
@@ -26,8 +26,48 @@ hub_tags:
   features: []
 sdk: 
   type: full 
-  version: 5 
+  version: 5.4.9 
   user: nobody
+links:
+ - "[Windows Defender ATP](https://www.microsoft.com/en-us/windowsforbusiness/windows-atp)"
+ - "[Windows Defender ATP API Start Page](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-atp/use-apis)"
+ - "[Windows Defender ATP API Endpoints](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-atp/exposed-apis-list)"
```

Comment:
Could we remove these from links and only keep in references? No point having overlap

Reply:
the links and references have been updated